opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -192,6 +198,15 @@ impl<R: RuntimeChannel> LogProcessor for BatchLogProcessor<R> {
     }
 
     fn shutdown(&self) -> LogResult<()> {
+        let dropped_logs = self.dropped_logs_count.load(Ordering::Relaxed);
+        if dropped_logs > 0 {
+            otel_warn!(
+                name: "BatchLogProcessor.Shutdown",
+                dropped_logs = dropped_logs,
+                message = "Logs were dropped due to a full buffer."
+            );
```

Comment:
would be nice to also include an attribute for the max queue size the batchprocessor was operating with.